non_www/tabs/school/download_region.html

```diff
@@ -61,13 +63,14 @@ <h4><a name="regschooldestinations"></a>School travel data for schools</h4>
      <div class="col-md-12 text-left" style="padding-top:10px;">
         <h4><a name="regschoolroutenetSchool"></a>School travel data for the route network</h4>
         <p>Download anonymised National School Census 2011 baseline and scenario cycling estimates for the route network in <!--region_name-->.  This was calculated using within-region flows (i.e. children who live and attend school in <!--region_name-->) containing at least 5 children.</p>
+        <p>Baseline and scenario data are not presented for route segments containing 1 or 2 cyclists.  Instead these data are set as missing ('NA').</p>
```

Comment:
Wow, the requirements are very strange.  So it lets us show that there is 1 or 2 but not say which?  Thanks for digging into this @AnnaGoodman1 !

Reply:
that is my understanding, see page 36 of the NPD user guide:

"NPD figure(s) at a local level must be based on *no fewer than 3 cases* or else suppressed (e.g. replaced with an X, with a note “* in this table means the figures have been suppressed due to small numbers”. *Zero is allowed*)"

https://www.gov.uk/government/uploads/system/uploads/attachment_data/file/472700/NPD_user_guide.pdf